syntax = "proto3";

package relation;

option go_package = ".proto/";

service RelationNetService{
  rpc QueryRelationNet (RelationRequest) returns (RelationResponse);
}

message RelationRequest{
  string username = 1;
}

message RelationResponse{
  repeated NetInfo netInfo = 4;
}

message NetInfo{
  string relation = 1;
  string username1 = 2;
  string username2 = 3;
}